README.md

```diff
@@ -1,6 +1,6 @@
-# Atoma Proxy infrastructure
+# Atoma Proxy Infrastructure
 
-<img src="https://github.com/atoma-network/atoma-node/blob/update-read-me/atoma-assets/atoma-pfp.jpg" alt="Logo" height="500"/>
+<img src="https://github.com/atoma-network/atoma-node/blob/ja-improve-readme/atoma-assets/atoma-banner.png" alt="Logo"/>
```

Comment:
is this `ja-improve-readme` the right branch for the image?

Reply:
Addressed